packages/app/src/cli/models/extensions/specifications.ts

```diff
@@ -20,8 +20,19 @@ export async function allThemeSpecifications(): Promise<ThemeExtensionSpec[]> {
 const memLoadSpecs = memoize(loadSpecs)
 
 async function loadSpecs(directoryName: string) {
-  const url = path.join(path.dirname(fileURLToPath(import.meta.url)), path.join(directoryName, '*.{js,ts}'))
-  let files = await path.glob(url, {ignore: ['**.d.ts', '**.test.ts']})
+  const url = path.join(path.dirname(fileURLToPath(import.meta.url)), directoryName)
+  /**
+   * When running tests, "await import('.../spec..ts')" is handled by Vitest which does
+   * transform the TS module into a JS one before loading it. Hence the inclusion of .ts
+   * in the list of files.
+   */
+  let files = (await path.glob(path.join(url, '*'))).filter((filePath) => {
+    const isTest = filePath.endsWith('.test.ts')
+    const isTypescript = filePath.endsWith('.ts')
+    const isDeclaration = filePath.endsWith('.d.ts')
+    const isJS = filePath.endsWith('.js')
+    return isJS || (isTypescript && !isDeclaration && !isTest)
+  })
```

Comment:
Another alternative, using the [dot option](https://github.com/mrmlnc/fast-glob#dot):

```suggestion
  const url = path.join(path.dirname(fileURLToPath(import.meta.url)), path.join(directoryName, '*.{js,ts}'))
  /**
   * When running tests, "await import('.../spec..ts')" is handled by Vitest which does
   * transform the TS module into a JS one before loading it. Hence the inclusion of .ts
   * in the list of files.
   */
  let files = await path.glob(url, {ignore: ['**.d.ts', '**.test.ts'], dot: true})
```